### PR TITLE
Fixed PaymentPrefillInfo of Trustly

### PR DESCRIPTION
**Pull request by aait**

### CODE REVIEW

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/SwedbankPay/Api/Service/Trustly/Resource/Request/PaymentPrefillInfo.php  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=6179903&bid=20170866)

File: src/SwedbankPay/Api/Service/Trustly/Resource/Request/PaymentPrefillInfo.php [hunk 1]
```diff
@@ -10,5 +10,45 @@
  */
 class PaymentPrefillInfo extends PrefillInfo implements PaymentPrefillInfoInterface
 {
+    /**
+     * Get Prefilled value to put in the first name text box.
+     *
+     * @return string|null
+     */
+    public function getFirstName()
```
Comment (asbjornu) [inline src/SwedbankPay/Api/Service/Trustly/Resource/Request/PaymentPrefillInfo.php:18]: Please add a test for `getFirstName`.

File: src/SwedbankPay/Api/Service/Trustly/Resource/Request/PaymentPrefillInfo.php [hunk 2]
```diff
@@ -10,5 +10,45 @@
...
+    {
+        return $this->offsetSet(self::FIRST_NAME, $firstName);
+    }
+
+    /**
+     * Get Prefilled value to put in the last name text box.
+     *
+     * @return string|null
+     */
+    public function getLastName()
```
Comment (asbjornu) [inline src/SwedbankPay/Api/Service/Trustly/Resource/Request/PaymentPrefillInfo.php:39]: Please add a test for `getLastName`.

**Review by asbjornu — CHANGES_REQUESTED** [review]

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tests/TrustlyPaymentPrefillInfoTest.php  1
- src/SwedbankPay/Api/Service/Trustly/Resource/Request/PaymentPrefillInfo.php  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=6179903&bid=20170866)

**Review by asbjornu — APPROVED** [review]

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tests/TrustlyPaymentPrefillInfoTest.php  1
- src/SwedbankPay/Api/Service/Trustly/Resource/Request/PaymentPrefillInfo.php  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=6179903&bid=20170866)

**Pull request merged**